cal/cal/static/js/app.js

```diff
@@ -3,32 +3,120 @@ var analyticsApp = angular.module('analyticsApp', ['nvd3', 'ui.calendar']);
 analyticsApp.controller('LoggedInCtrl', function LoggedInController($scope) {
 });
 
-// Controller to generate graph data from chart directive, cumulative tags
 analyticsApp.controller('TagsCtrl', function($scope, $http){
   var url = '/v1/tags.json';
+  var tagUrl = '/v1/tags/';
```

Comment:
Actually, is `/v1/tags/.json` the same as `/v1/tags.json`? If so, then I think it would be better to have `tagUrl` be `/v1/tags/`. so that we don't have to do `tagUrl + '/' + id`